src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,11 +9,15 @@ public class CliSyntax {
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_MODULE = new Prefix("m/");
     public static final Prefix PREFIX_VENUE = new Prefix("v/");
     public static final Prefix PREFIX_TIMESLOT = new Prefix("t/");
```

Comment:
Lets change timeslot to some other alternatives as a temporary solution so that its diff from tags